content/operate/guides/cosmos/syncing.md

```diff
@@ -4,6 +4,236 @@ title: Syncing
 showH2InSideNav: true
 ---
 
-In this guide, we'll show you how to use [Firehose](/operate/concepts/) to sync and stream Cosmos chains.
+Below, we'll show you how to use [Firehose](/operate/concepts/) to sync and stream Cosmos Chains.
 
-_Coming Soon_
+---
+
+## Install firehose-cosmos
+
+`firehose-cosmos`, is an application that runs a few small, isolated processes,
+that together form the `Firehose` stack. A thorough discussion of the [Concepts & Architecture]({{< ref "/operate/concepts" >}})
+is discussed elsewhere. Needless to say, you must run `firehose-cosmos` to run a `Firehose` locally.
+
+You can download the latest version of `firehose-cosmos` [here](https://github.com/figment-networks/firehose-cosmos/releases),
+however we recommend you use the Dockerfile provided in the `firehose-cosmos` repository located [here](https://github.com/figment-networks/firehose-cosmos).
+
+As of the time of writing this, the latest release is v0.4.0, you can get a copy of this from [Figment's Docker Hub](https://hub.docker.com/r/figmentnetworks/firehose-cosmos/tags). The latest release will always be listed on the [Github Releases Page](https://github.com/figment-networks/firehose-cosmos/releases). Ensure you always use the most up to date version of `firehose-cosmos` to ensure you have the latest functionality.
+
+```bash
+docker run --rm -it figmentnetworks/firehose-cosmos:0.4.0 /app/firehose help
+```
+
+Alternatively you can clone the repository and run `make install` to install it locally.
+
+To verify the installation was successful, run:
+
+```bash
+firehose-cosmos --version
+# or
+docker run --rm -it figmentnetworks/firehose-cosmos:0.4.0 /app/firehose --version
+```
+---
+
+## Install and run instrumented nodes
+
+In order to index Cosmos nodes with `firehose-cosmos`, you will need to insure you are using the correctly modified binaries for the chain you are targetting. Figment provide these for CosmosHub and Osmosis, mainnet and testnet chains as prebuilt Docker Images which you can find listed on [their Dockerhub](https://hub.docker.com/r/figmentnetworks/firehose-cosmos/tags). These Dockerfiles also start the `firehose-cosmos` process for you. Below is an example for v7.0.4 of the Osmosis testnet. You can modify this to use whichever version of the chain node you are indexing.
+
+```bash
+docker run --rm -it figmentnetworks/firehose-cosmos:fh-v0.4.0-osmosis1-testnet-v7.0.4 /app/firehose start
```

Comment:
Maybe you should recommend using a volume mount (`-v /my/data:/fh-data/storage` or similar) at this point, and give a few words on what will be stored there (the instrumented node data, firehose blocks) @lavabyrd  ?
apart from that, LGTM